codesystems/CodeSystem-England-FGMRemovalReason.xml

```diff
@@ -0,0 +1,36 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-FGMRemovalReason" />
+  <url value="https://fhir.nhs.uk/CodeSystem/England-FGMRemovalReason" />
+  <identifier>
+    <system value="https://fhir.nhs.uk/identifier/CodeSystem" />
+    <value value="fgm-removal-reason" />
+  </identifier>
+  <version value="1.0.0" />
+  <name value="EnglandFGMRemovalReason" />
+  <title value="England FGM Removal Reason" />
+  <status value="draft" />
+  <experimental value="true" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A codesystem used for the removal of FGM Flag" />
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <caseSensitive value="true" />
+  <content value="complete" />
+  <concept>
+    <code value="0"></code>
+    <display value="Added in error - this child does not have a family history of FGM" />
```

Comment:
Why would an FGM flag be removed in the context of *family history* instead of just the child?

Reply:
Required by the current Spine implementation.
DN: We will have to make sure the review covers the question on 'purpose of API' - is it 'risk of' or 'family history of', is that clear and consistent in the codes, and in the IG text?